tests/writer.rs

```diff
@@ -310,6 +311,7 @@ mod tests {
             datatype: NiftiType::Rgb24 as i16,
             pixdim: [1.0; 8],
             sform_code: 2,
+            qform_code: 0,
```

Comment:
> It seems that the writer does behave differently

You made me worried with this. It's only because we compare the image bytes by bytes. Changing the right byte like we do in lines 287-293 or changing it for real in the RGB image on disk should work too. However, keeping `qform_code: 0,` is simpler. As you wish.

Reply:
Thank you for clearing this out. Please let me know if you think this test should be made more transparent of this matter.

Reply:
You forgot to remove `pixdim` ;)

And, yes, it's probably a good idea to add a comment in `rgb_header_gt` explaining why `qform_code: 0,` otherwise we will forget soon enough. Or if you have a hex editor, just change the byte from 0 to 1 (I can also do it in another branch, you don't have to do everything).

Reply:
Oops. Hopefully this is it.